### ISSUE TITLE
Jade on Wine on Ubuntu 16.04 causes apparent X Window freeze --- have to reboot 

### ISSUE THREAD

**Issue by dalehenrich** [jgfoster/Jade.bak#135]
This happened twice in a row (within 5 minutes of opening up Jade) when I was attempting to create a new Monticello package and save  it to a FileTree repository. 

I assume that this issue is somehow related to using Wine on Ubuntu. 

Is there a key combination that I can use to kill Jade on Linux? 

The entire X display is basically unresponsive ... the mouse is active and the launcher menu unfolds, but it immediately hides as if it did not get focus ... clicking all around the display yields nothing useful ... Jade turns dark with the menu "highlighted" ... something I've seen with thunderbird, but thunderbird has not captured the mouse and the rest of the x display is not frozen ...

**Comment by jgfoster** [comment]
I haven't used Jade on Linux. On the Mac it uses X but I haven't experience any lockups. To the extent that the entire X display is unresponsive, can you get in using a shell (e.g., SSH)? If so, then can you kill Wine? The X client? It seems surprising that Linux would be unresponsive due to an X Windows lockup.

**Comment by dalehenrich** [comment]
I'm on my laptop at home, so when it locks up I have no choice but to reboot ... Many things in Jade appear to work properly but it seems that the same menu is the culprit for both lockups, however. I'm not familiar enough with Jade to know if it is the same menu or one that looks similar --- after the lockup I can't take a screenshot:)

**Comment by jgfoster** [comment]
Take a picture with your phone!

**Comment by dalehenrich** [comment]
I used gemtools to verify that Monticello was working ...  when I reach a point where I can afford to reboot my machine I will make another attempt ... for now I will avoid using Jade on a linux box...

**Comment by IwanVosloo** [comment]
I have been experimenting with Jade under wine on Ubuntu 16.04 a bit, and do not experience such lockups. I DID get issues that looked like timeouts: sometimes programs would start only after having waited for 20-30 seconds and certain actions in Jade behaved similarly.  (This only happened inside an lxd container.)

I could reproduce this using only standard wine programs, like winecfg, so my issues were not Jade-specific.

My problems were solved though after I installed wine 3.0 from the apt repositories at winehq. The version in ubuntu 16.04 is quite old. 

I think this was probably a slightly different problem, but perhaps this solution is also worth a try for other wine issues on 16.04?
